l10n_ch_hr_payroll/models/hr_employee.py

```diff
@@ -5,7 +5,7 @@
 #
 #  Created by sge@open-net.ch
 #
-#  Copyright (c) 2014 Open-Net Ltd.
+#  Copyright (c) 2015 Open-Net Ltd.
```

Comment:
You can't do that, copyright should keep creation year of the file.
2014-2015 would be right if you changed that file this year.

Or was this file created in 2015?